Makefile

```diff
@@ -43,13 +43,16 @@ install:
 	@echo "> Installing sources, dependencies and database."
 	-${remote} "git clone https://github.com/cafebabel/cafebabel.com.git ${src_dir}"
 	-${remote} "python3.6 -m venv ${venv_dir}"
-	-${remote} "${goto_src} && make make_dirs"
+	-${remote} "${goto_src} && make make-dirs"
 	make reset-db
 	make deploy
 
-make_dirs:
+make-dirs:
 	mkdir -p ./logs
 
 reset-db:
-	${remote} "${goto_src} && FLASK_APP=prod flask initdb"
-	${remote} "${goto_src} && FLASK_APP=prod flask load_fixtures"
+	make flask-command command=initdb
+	make flask-command command=load_fixtures
+
+flask-command:
+	${remote} "${goto_src} && FLASK_APP=prod flask $(command)"
```

Comment:
We [use _preprod_](https://github.com/cafebabel/cafebabel.com/pull/336/commits/5bb9a4e68e06cfbd4a971014ed2fd638b5efc468#diff-b67911656ef5d18c4ae36cb6741b7965R32), but that is changing to `${env}` anyways in a next PR

Reply:
What is the difference between parenthesis and brackets?